### PR TITLE
chore: upgrade vite version

### PR DESCRIPTION
**Pull request by sudongyuer**
#1489

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,8 +1,7 @@
-lockfileVersion: 5.4
```
Comment (antfu) [inline pnpm-lock.yaml:1]: Please enable corepack

**Review by antfu — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml (hunk 1 above)
Comment (sudongyuer) [reply pnpm-lock.yaml]: Do i solve it tony ?

**Review by antfu — APPROVED** [review]

**Comment by antfu** [comment]
The test is failing. I guess the override is still needed to make sure a single version of Vite installed.

**Comment by sudongyuer** [comment]
> 

#1505

**Pull request closed without merging**